src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,27 @@
+Exercises: The Early Bird Gets the ORM!
+=======================================
+
+Now that we have gotten very familiar with our ``CodingEvents`` application, let's design some additional features.
+As you work on your ``CodingEvents`` application, you may have been inspired by `Meetup <https://www.meetup.com/>`_!
```

Comment:
can we remove either the "!" from the title or this statement -too many too close diminishes their worth

Reply:
ok actually there a 5 ! on this short page. id argue we can remove them all